.spelling

```diff
@@ -912,6 +912,24 @@ bs
 # To allow Site Init
 Init
 
+- operations/boot_orchestration/Determine_Which_BOS_Session_Booted_A_Node.md
+bos_session_id
+ssh
+cray
+bos
+BOS_SESSION_ID
+json
+templateName
+knn-boot-x3000c0s28b4n0
+jq
+x3000c0s17b0n0
+94e712ab-df76-40ee-8cfb-7ac487fd8a13
+2023-06-27T00
+end_time
+start_time
+template_name
+gdr-tmpl
+
```

Comment:
You shouldn't need any of these if you put your commands and command output in code blocks properly as suggested in my other comments.

```suggestion
```